lib/tests/lib.rs

```diff
@@ -25,15 +17,39 @@ fn set_target_actor(tester: &mut Tester, name: String, binary: Vec<u8>, abi: Abi
 }
 
 #[test]
-fn test_tester_test() {
+fn test_deploy_non_valid_target_actor() {
+    // Instantiate tester
+    let mut tester = Tester::new();
+
+    let target_actor = WasmActor::new(
+        String::from("Target.wasm"),
+        vec![1, 2, 3],
+        Abi {
+            constructor: None,
+            set_up: None,
+            methods: vec![Method::new_from_name("Hello").unwrap()],
+        },
+    );
+
+    let res = tester.deploy_target_actor(target_actor);
+    assert!(res
+        .err()
+        .as_ref()
+        .unwrap()
+        .to_string()
+        .contains("Non valid target actor wasm file: Target.wasm"))
```

Comment:
I'd say that if we really want to be sure that this an error for this method we could create an `Error` variant for it